src/ui/scss/component/_icon.scss

```diff
@@ -0,0 +1,10 @@
+// Not all icons are created equally... at least the react-feather ones aren't
+// Minor adjustments to ensure icons line up vertically
+
+.icon--Heart {
+  top: -1px;
```

Comment:
How will this scale when icons are rendered at different sizes?

Reply:
That's a good point. This won't scale well. I think it will work if we use `em`. Currently our icons are all locked in at the same size, regardless of font size. Will keep that in mind going forward.